### PR TITLE
Fix: Webpack options

### PR DESCRIPTION
**Pull request by xhulz**
## PR description

The`devtool` option was changed from `source-map` to `eval`. We know that this may not be the best option, however, the memory of the virtual machines was overflowing due to the large use of resources to generate the code mapping.

## Documentation

- [ ] I thought about documentation and added the `doc-change-required` label to this PR if documentation updates are required.

### CODE REVIEW

**Review by acuarica — APPROVED** [review]
lgtm

**Comment by acuarica** [comment]
Hey @xhulz, thanks for taking care and sending this. Do you know why this is happening all of a sudden? Is it something coming from Truffle upstream?

**Comment by kevinbluer** [comment]
Looks good from my side too! As an additional check, I'll do smoke test off the pre-release when it's published to the marketplace 👍 Thanks for the wrangling on this @xhulz!

**Comment by acuarica** [comment]
Thanks for the detail explanation @xhulz, super helpful!

**Comment by acuarica** [comment]
Hey, related to 
- https://github.com/trufflesuite/vscode-ext/issues/131

I've running some benchmarks to see if we can improve performance. In the list below you can see the command and the size of its output, except in the 4th command, which shows an approximation of the _Activation time_.

### On develop

- **`yarn webpack:dev && ll out/src`**. `[..] 79M [..] extension.js`
- **`yarn webpack:prod && ll out/src`**. `[..] 31M [..] extension.js`
- **`yarn package`**. `DONE  Packaged: [..]/truffle-vscode-2.3.5.vsix (110 files, 10.38MB)`
- **`code --install-extension truffle-vscode-2.3.5.vsix`**. _Activation time 769ms_

### With [no `faker`](https://github.com/trufflesuite/vscode-ext/pull/264)

- **`yarn webpack:dev && ll out/src`**. `[..] 70M [..] extension.js`
- **`yarn webpack:prod && ll out/src`**. `[..] 28M [..] extension.js`
- **`yarn package`**. `DONE  Packaged: [..]/truffle-vscode-2.3.5.vsix (110 files, 9.67MB)`
- **`code --install-extension truffle-vscode-2.3.5.vsix`**. _Activation time 713ms_

### With [no `faker`](https://github.com/trufflesuite/vscode-ext/pull/264) and [`devtool: false`](https://webpack.js.org/configuration/devtool/) in `webpack.prod.js`

- **`yarn webpack:prod && ll out/src`**. `[..] 15M [..] extension.js`
- **`yarn package`**. `DONE  Packaged: [..]/truffle-vscode-2.3.5.vsix (110 files, 7.6MB)`
- **`code --install-extension truffle-vscode-2.3.5.vsix`**. _Activation time 529ms_

This shows that by removing `faker` we improve a little bit, but by far the major improvement is when we remove `devtool`.
So in summary, should we remove `devtool`, _i.e._, `devtool: false`, from the production bundle?

**Comment by xhulz** [comment]
Hey @acuarica,

Thanks for that :)

I've tried to bundling using `devtool: false` but I've got some problems to do that on Windows. Windows shut down the process every time. Have you tried creating a VSIX on Windows VM with the same Github CI setup?

**Comment by acuarica** [comment]
Not yet, but I was wondering about any potential issues we might have when removing `devtool` in the production bundle. Do you know why we had `source-map` in the first place? Maybe @michaeljohnbennett or @kevinbluer know about this.

**Comment by xhulz** [comment]
No, i don't...

**Comment by acuarica** [comment]
> Have you tried creating a VSIX on Windows VM with the same Github CI setup?

No, so let's see https://github.com/trufflesuite/vscode-ext/pull/268

**Comment by michaeljohnbennett** [comment]
Hey, not sure about the source-map, I think it was probably a historical thing, but in reality we don't need it for debugging the packed plugin so if you can get it working to deploy and run without it, do it! 👍🏻
